### ISSUE TITLE
RxJS 2.4.7+ "clearMethod is not defined"

### ISSUE THREAD

**Issue by metrofun** [Reactive-Extensions/RxJS#650]
I'm receiving "clearMethod is not defined" inside `scheduleNow`

**Comment by ntkog** [comment]
Me too, but in v2.4.8:

My environment is : 

MacBook Pro (Retina, 15-inch, Late 2013) : Yosemite : 10.10.1
```bash
$node --version
v0.12.0
$npm ls
└── rx@2.4.8
```

I've tried with the example shown in [Documentation]

```js
// Generate a value with an absolute time with an offset of 100ms multipled by value
var source = Rx.Observable.generateWithRelativeTime(
    1,
    function (x) { return x < 4; },
    function (x) { return x + 1; },
    function (x) { return x; },
    function (x) { return 100 * x; }
).timeInterval();

var subscription = source.subscribe(
    function (x) {
        console.log('Next: ' + x);
    },
    function (err) {
        console.log('Error: ' + err);
    },
    function () {
        console.log('Completed');
    });

// => Next: {value: 1, interval: 100}
// => Next: {value: 2, interval: 200}
// => Next: {value: 3, interval: 300}
// => Completed
``` 

The error : 

```bash
ReferenceError: clearMethod is not defined
    at null.action (/*******/node_modules/rx/dist/rx.js:1420:9)
    at Disposable.dispose (/********/node_modules/rx/dist/rx.js:754:12)
    at CompositeDisposablePrototype.dispose (/*********/node_modules/rx/dist/rx.js:737:31)
    at BooleanDisposable.Rx.SingleAssignmentDisposable.booleanDisposablePrototype.dispose (/********/node_modules/rx/dist/rx.js:823:18)
    at AutoDetachObserverPrototype.dispose (/********/node_modules/rx/dist/rx.js:4915:14)
    at BooleanDisposable.Rx.SingleAssignmentDisposable.booleanDisposablePrototype.dispose (/*******/node_modules/rx/dist/rx.js:823:18)
    at AutoDetachObserverPrototype.dispose (/*********/node_modules/rx/dist/rx.js:4915:14)
    at BooleanDisposable.Rx.SingleAssignmentDisposable.booleanDisposablePrototype.dispose (/*******/node_modules/rx/dist/rx.js:823:18)
    at AutoDetachObserverPrototype.dispose (/*********/node_modules/rx/dist/rx.js:4915:14)
    at AutoDetachObserverPrototype.completed (/********/node_modules/rx/dist/rx.js:4906:12)
```


[Documentation]:https://github.com/Reactive-Extensions/RxJS/blob/master/doc/api/core/operators/generatewithrelativetime.md

**Comment by mattpodwysocki** [comment]
@metrofun fixed in 2.4.9

**Comment by metrofun** [comment]
@mattpodwysocki That was fast, thx :horse_racing:
